rebar.config

```diff
@@ -56,9 +56,9 @@
     {sext, {git, "git://github.com/uwiger/sext.git", {tag, "1.4.1"}}},
     {riak_pipe, {git, "git://github.com/basho/riak_pipe.git", {tag, "riak_kv-3.0.5"}}},
     {riak_dt, {git, "git://github.com/basho/riak_dt.git", {tag, "riak_kv-3.0.0"}}},
-    {riak_api, {git, "git://github.com/basho/riak_api.git", {tag, "riak_kv-3.0.5"}}},
+    {riak_api, {git, "git://github.com/basho/riak_api.git", {branch, "mas-i1788-reconcileapi"}}},
```

Comment:
A little to-do after update of sub projects to point to new tag

Reply:
Yes, will pick this up as part of the release cycle